### ISSUE TITLE
Dynamic import tracing bug

### ISSUE THREAD

**Issue by guybedford** [jspm/jspm-cli#2475]
Currently `import('asdf')` is failing `jspm trace` due to it only supporting `"` strings!

Fix included in https://github.com/jspm/jspm-cli/commit/a8a307f758c1f2737c491ae88335ccd73d781aa8.

**Comment by irustm** [comment]
that's gonna fix this error?
```
> jspm map ./lib/main.js -o importmap.json 
err  Error: Unable to resolve bare specifier "'" from /jspm_packages/npm/@angular/core@7.2.15/bundles/core.umd.dew.js
```

**Comment by guybedford** [comment]
Yes, I'm posting a release shortly.

**Comment by guybedford** [comment]
Released in 2.0.0-beta.4.
